apps/src/constants.js

```diff
@@ -57,6 +58,7 @@ exports.TestResults = {
 
   // Codes for unvalidated levels.
   UNSUBMITTED_ATTEMPT: -50,           // Progress was saved without submitting for review, or was unsubmitted.
+  UNVALIDATED_SUBLEVEL: -49,          // Either a member of a LevelGroup or a contained level, for which the result has not been validated
```

Comment:
I think the client will cache "larger" values (where any value takes precedence over zero), so it may be worth some thought about if this will ever conflict/interact with the existing values.